### PR TITLE
hello Microverse 2: linterns, html, css, gitignore

### PR DESCRIPTION
**Pull request by Wikitelmex**
## Modifications
-Added lintern.yml file to checkouts
-index.html file added with a header
-style.css file added and added some header styles
.gitignore updated to avoid module folder

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
```
Comment (algerina) [inline readme.md]: Hi

File: htmlcss-lintern.json [hunk 2]
```diff
@@ -0,0 +1 @@
+some lintern code
```
Comment (Wikitelmex) [inline htmlcss-lintern.json:1]: add real code to lintern

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (ahmetbozaci) [inline readme.md]: - [ ]  You should update your README file. You can use this [template](https://github.com/microverseinc/readme-template) and make sure there are no  broken links(example screenshot, licence) and empty sections. If you won't use some sections, delete them. 
- [ ] I recommend using as many readme sections as you can.
- [ ] You should change the readme template's sentence with your sentence.

File: index.html [hunk 4]
```diff
@@ -0,0 +1 @@
+<h1>Hello Microverse</h1>
```
Comment (ahmetbozaci) [inline index.html]: - [ ] You should  add some style to this tag. [Requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=A%20.css%20file%20that%20includes%20one%20class%20with%20styles%20for%20your%20header%20(make%20it%20in%20your%20favorite%20color).)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️

### Hello @Wikitelmex ✋🏼  Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

### 💯  Nice to have 💯
 
- Your git flow is correct ✔️
- You create html and css files according to the requirement.✔️

### ♻️ Request changes ♻️
- [ ] Add a short summary of changes introduced in the pull request. Please, check this [lesson in curriculum](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
- [ ] You didn't set up linter files. Please check this [repository](https://github.com/microverseinc/linters-config/tree/master/html-css) and set up linter files correctly.  Make sure all linter tests pass.
Screenshot from 2021-08-10 20-01-09



- [ ] _**Check the comments under the review.**_


### OPTIONAL SUGGESTIONS

#### These are optional but highly recommended.

- [ ] Your commit messages should always start  with capital letters and you should write an imperative tense. I see some of them are not like that. You can change them by using `git rebase`. 
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)


### Cheers and Happy coding! 💻 👏
almost

___
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: readme.md (hunk 3 above)
Comment (Wikitelmex) [reply readme.md]: > # STATUS: Changes Required ♻️ -first
> ### Hello @Wikitelmex ✋🏼 Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.
> ### 💯 Nice to have 💯
> * Your git flow is correct ✔️
> * You create html and css files according to the requirement.✔️
> 
> ### ♻️ Request changes ♻️
> * [ ]  Add a short summary of changes introduced in the pull request. Please, check this [lesson in curriculum](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
> * [ ]  You didn't set up linter files. Please check this [repository](https://github.com/microverseinc/linters-config/tree/master/html-css) and set up linter files correctly.  Make sure all linter tests pass.
>   Screenshot from 2021-08-10 20-01-09
> * [ ]  _**Check the comments under the review.**_
> 
> ### OPTIONAL SUGGESTIONS
> #### These are optional but highly recommended.
> * [ ]  Your commit messages should always start  with capital letters and you should write an imperative tense. I see some of them are not like that. You can change them by using `git rebase`.
> * [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
> * [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
> 
> ### Cheers and Happy coding! 💻 👏
> almost
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
> 
> _If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [ahmetbozaci](https://github.com/ahmetbozaci), I will receive a notification and will respond._
> 
> [GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)
> 
> _Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

I have some issues running the linterns here on Github, looks like the server don't response, but, the lintern file added as the tutorial says.

Comment (Wikitelmex) [reply readme.md]: I have some issues running the linterns here on Github, looks like the server doesn't respond, but, the lintern file was added as the tutorial says.

Updated the README file

File: index.html (hunk 4 above)
Comment (Wikitelmex) [reply index.html]: style added in style.css as a class and implemented inside the index.html h1 tag

**Review by barackm — APPROVED** [review]
Hi @Wikitelmex , 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please it's good to remove the node_modules folder from the repository and add it to the .gitignore file because it's a very big folder and it can get regenerated by running `npm install `. this makes the review so difficult as well.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Wikitelmex** [comment]
@barackm thank you very much, I already merged, I deleted module folder from github repo, is anything else that I must to do? because I still have the yellow bag in my course. Thanks again

image

**Comment by barackm** [comment]
Hi @Wikitelmex, now as the badge says `submit` you can submit the project, if you click on actions you will find an option  `submit` click on it and then fill out the form with the information they will request.

**Pull request merged**